packages/office-ui-fabric-react/src/components/Pivot/Pivot.Props.ts

```diff
@@ -50,6 +50,18 @@ export interface IPivotProps extends React.Props<Pivot> {
    */
   linkFormat?: PivotLinkFormat;
 
+  /**
+   * Specify whether to skip rendering the tabpanel with the content of the selected tab.
+   * Use this prop if you plan to separately render the tab content
+   * and don't want to leave an empty tabpanel in the page that may confuse Screen Readers.
+   */
+  headersOnly?: boolean;
```

Comment:
Yep, I started thinking like you, but then I realized that using the `Pivot` without content was trivial and I did it with empty `PivotItem`s successfully.
Next we realized we didn't complain with ARIA spec, and that's the goal of this PR.

So my opinion is that we don't need extra complexity, we can just demonstrate with the example how this can be easily achieved.